lib/database/adapters/RunAdapter.js

```diff
@@ -11,6 +11,17 @@
  * or submit itself to any jurisdiction.
  */
 
+const LHC_REVOLUTION = 11245; // The LHC revolution frequency: 11.245kHz (~ c/26.7km).
```

Comment:
Comment is not the same as variable, it might be renamed to LHC_REVOLUTION_FREQUENCY_HZ

Reply:
applied